lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -208,9 +212,11 @@ class ReportUIFeatures {
   _setupToolsButton() {
     this.toolsButton = this._dom.find('.lh-tools__button', this._document);
     this.toolsButton.addEventListener('click', this.onToolsButtonClick);
+    this.toolsButton.addEventListener('keydown', this.onToolsButtonKeydown);
```

Comment:
is keyup not appropriate here?

Reply:
Apologies for not replying earlier to this comment. I tried keyup and found that it would also cause a scroll of the report document with the UP/Down arrow keys.